src/Fixer/ControlStructure/NoUnneededCurlyBracesFixer.php

```diff
@@ -116,6 +116,9 @@ private function clearOverCompleteBraces(Tokens $tokens, int $openIndex, int $cl
         $tokens->clearTokenAndMergeSurroundingWhitespace($openIndex);
     }
 
+    /**
+     * @return iterable<int>
```

Comment:
```suggestion
     * @return int[]
```

would there by any difference?

Reply:
ah, so `T[]` implies array and not iterable? got it

Reply:
hmm, well, any `array` is `iterable`, so `T[]` is as well `iterable`, 
I'm just saying that `iterable<int>` is a smaller type as it doesn't state it is an `array`.

Reply:
I was wrongly assuming that `T[]` giving `iterable`, now I see it's giving `array`. thanks!